packages/thirdweb/src/wallets/in-app/native/helpers/auth/middleware.ts

```diff
@@ -13,12 +13,12 @@ import { setUpShareForNewDevice } from "../wallet/retrieval.js";
 export async function postAuth({
   storedToken,
   client,
-  recoveryCode,
+  encryptionKey,
   storage,
 }: {
   storedToken: AuthStoredTokenWithCookieReturnType["storedToken"];
   client: ThirdwebClient;
-  recoveryCode?: string;
+  encryptionKey?: string;
```

Comment:
Do we still want/need this? Prevents custom auth from being used with ecosystem wallets